resolver-dns-native-macos/src/test/java/io/netty/resolver/dns/macos/MacOSDnsServerAddressStreamProviderTest.java

```diff
@@ -18,24 +18,21 @@
 import io.netty.resolver.dns.DnsServerAddressStream;
 import io.netty.resolver.dns.DnsServerAddressStreamProvider;
 import io.netty.resolver.dns.DnsServerAddressStreamProviders;
-import org.junit.Assume;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.instanceOf;
-import static org.junit.Assert.assertNotEquals;
-import static org.junit.Assert.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
 
 public class MacOSDnsServerAddressStreamProviderTest {
 
-    @BeforeClass
-    public static void assume() {
-        Assume.assumeTrue(MacOSDnsServerAddressStreamProvider.isAvailable());
-    }
-
     @Test
+    @EnabledOnOs(OS.MAC)
```

Comment:
Should we add `@EnabledOnOs` directly on the class level

```
@Target({ElementType.TYPE, ElementType.METHOD})
...
public @interface EnabledOnOs {
```

nit: With JUnit 5 you can remove the public modifiers